src/components/inputs/answers/SelectInput.jsx

```diff
@@ -0,0 +1,35 @@
+import { React } from 'react';
+
+const styles = `
+    .font-barlow {
+        font-family: 'Barlow', sans-serif;
+    }
+`;
+
+function SelectInput(props) {
+    console.log(props);
+    const options = props.input.options || [];
+    return (
+        <div className="rounded shadow bg-white font-barlow pt-3 px-0 mt-3">
+            <div className="d-flex column align-items-center justify-content-between m-0 pb-3">
+                <div className="d-flex col-9 align-items-center text-break text-start px-3">
+                    {props.input.question}
+                </div>
+                <div className="px-0">
```

Comment:
```suggestion
                <div className="col px-0">
```
- Aparentemente o framework inferiu tratar-se de uma coluna, mas convém classificar;

Reply:
ta feito chefe